pkgs/shells/fish/default.nix

```
@@ -271,14 +295,37 @@ let
 
     doCheck = true;
 
-    nativeCheckInputs = [
-      coreutils
-      (python3.withPackages (ps: [ ps.pexpect ]))
-      procps
+    nativeCheckInputs =
+      [
+        coreutils
+        glibcLocales
+        (python3.withPackages (ps: [ ps.pexpect ]))
+        procps
+      ]
+      ++ lib.optionals stdenv.hostPlatform.isDarwin [
+        # For the getconf command
+        darwin.system_cmds
+      ];
+
+    checkTarget = "test";
+    preCheck = ''
+      export TERMINFO="${ncurses}/share/terminfo"
+    '';
```

```suggestion
    preCheck = ''
      export TERMINFO="${ncurses}/share/terminfo"
      runHook checkPhase
    '';
```

This provides a best practice in nix by running the checkPhase

---

Nvm ignore this.